ignite/handlers/param_scheduler.py

```diff
@@ -287,7 +287,8 @@ class CyclicalScheduler(ParamScheduler):
         If the scheduler is bound to an 'ITERATION_*' event, 'cycle_size' should
         usually be the number of batches in an epoch.
 
-    .. versionadded:: 0.4.5
+    .. versionchanged:: 0.4.13
+        Added warmup to the scheduler using ``warmup_each_cycle`` and ``warmup_duration``.
```

Comment:
We should keep previous tag:
```suggestion
.. versionadded:: 0.4.5

.. versionchanged:: 0.4.13
        Added warm-up option to the scheduler using ``warmup_each_cycle`` and ``warmup_duration``.
```